<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsWindow</class>
 <widget class="QMainWindow" name="GpsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Package date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLCDNumber" name="_dataPacote">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Package time:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLCDNumber" name="_horaPacote">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Latitude:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="_latitude">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Longitude:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="_longitude">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Speed in KM/H:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="_velocidade">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Altitude:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLCDNumber" name="_altitude">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Number of Satellites:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="_numeroSatelites">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Horizontal Precison:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLCDNumber" name="_precisaoPacote">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>10</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QTextEdit" name="_console">
      <property name="styleSheet">
       <string notr="true">
	background-color: black;
	color: lightgreen;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QTextEdit" name="_sateliteInfo"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QListWidget" name="_sateliteList"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSTART_FROM_SERIAL"/>
   <addaction name="actionSTOP"/>
   <addaction name="actionQUIT"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQUIT">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSTART_FROM_SERIAL">
   <property name="text">
    <string>SerialSetting</string>
   </property>
  </action>
  <action name="actionSTOP">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
